util/libcrypto.num

```diff
@@ -5876,3 +5876,4 @@ i2d_OSSL_ALLOWED_ATTRIBUTES_SYNTAX      ?	3_5_0	EXIST::FUNCTION:
 OSSL_ALLOWED_ATTRIBUTES_SYNTAX_free     ?	3_5_0	EXIST::FUNCTION:
 OSSL_ALLOWED_ATTRIBUTES_SYNTAX_new      ?	3_5_0	EXIST::FUNCTION:
 OSSL_ALLOWED_ATTRIBUTES_SYNTAX_it       ?	3_5_0	EXIST::FUNCTION:
+X509_STORE_CTX_set0_ocsp_resp           ?	3_5_0	EXIST::FUNCTION:OCSP
```

Comment:
Please rebase to fix the merge conflict.

You can avoid this trivial type of conflict in the future by adding the new function not at the end, but anywhere more up in the list of `3_5_0` entries.